99.wc_object/wc.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'optparse'
+require_relative 'wc_file'
+
+class Wc
+  def initialize(argv)
+    @filenames = argv
+    @options = {}
+    extract_options
+  end
+
+  def display
+    unless all_files_exist?
+      puts '指定のファイルは存在しません'
+      return
+    end
+
+    create_result_texts.each do |file_stat_text|
+      puts file_stat_text
+    end
```

Comment:
```suggestion
    puts create_result_texts
```

配列の内容を出力するだけなら↑でも同じ結果になりそうです

Reply:
修正しました(そういえばそういうこともできましたね....)。